core/controller/src/main/scala/whisk/core/entitlement/Entitlement.scala

```diff
@@ -225,12 +247,17 @@ protected[core] abstract class EntitlementProvider(config: WhiskConfig, loadBala
         }.map { _.forall(identity) }
     }
 
-    /** Limits activations if the load balancer is overloaded. */
-    protected def checkSystemOverload(subject: Subject, right: Privilege)(implicit transid: TransactionId) = {
+    /**
+     * Limits activations if the system is overloaded.
+     *
+     * @param right the privilege, if ACTIVATE then check quota else return None
+     * @return None if system is not overloaded else a rejection
+     */
+    protected def checkSystemOverload(right: Privilege)(implicit transid: TransactionId) = {
```

Comment:
The happiness of my reviewer(s) is very important to me. https://github.com/openwhisk/openwhisk/pull/1613/commits/9865aa5aff49b7ebf5ad8873f433f759a549a4f8